src/main/java/com/algorithmtron/controller/SortCharsByFrequencyController.java

```diff
@@ -0,0 +1,34 @@
+package com.algorithmtron.controller;
+
+import com.algorithmtron.model.StringDTO;
+import com.algorithmtron.service.SortCharsByFrequencyService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@Controller
+public class SortCharsByFrequencyController {
+
+    @Autowired
+    SortCharsByFrequencyService sortCharsByFrequencyService;
+
+    @GetMapping("/SortCharsByFrequency")
+    public String getSortCharsByFrequency(Model model) {
+        StringDTO stringDTO = new StringDTO();
+
+        model.addAttribute("stringDTO", stringDTO);
+
+        return "SortCharsByFrequencyInputPage";
```

Comment:
Does this need to be with capital letter?

Reply:
you're right! it slipped my eyes. will change. :)